analyzer/vulnerability-consumer/src/main/java/eu/fasten/analyzer/vulnerabilityconsumer/utils/Vulnerability.java

```diff
@@ -141,6 +144,18 @@ public Double getScoreCVSS3() {
         return scoreCVSS3;
     }
 
+    public String getVectorCVSS2() {
```

Comment:
Does this new extra information get injected into the metadata field of packages?

Reply:
Yes, it will be contained in each Vulnerability Statement that we store.